lib/pure/complex.nim

```diff
@@ -376,6 +376,15 @@ proc `$`*(z: Complex): string =
 
 {.pop.}
 
+proc FFT*(dataSource : seq[float]):seq[Complex]= 
```

Comment:
Dunno, I don't see much point in including this in the *standard* library. People who are serious about FFT will use a much better (=efficient) optimized implementation anyway, and the choice of FFT algorithm is too application specific. I don't think this belongs to the standard library, just as various geometry intersection routines don't belong to the standard library either.

However, you could create a 3rd party FFT module for this and maybe add some better performing variants as well.